### PR TITLE
Autocorrect RSpec/LeadingSubject RuboCop offenses, part I

### PR DESCRIPTION
**Pull request by eduardoj**
We have almost 400 offenses of this kind. I split fixing the offenses in several pull requests, to make it easier to review the changes proposed.

### CODE REVIEW

File: src/api/spec/policies/event_subscription/form_policy_spec.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
 RSpec.describe EventSubscription::FormPolicy do
+  subject { described_class }
```
Comment (hennevogel) [inline src/api/spec/policies/event_subscription/form_policy_spec.rb:2]: This is even already there implicitly

File: src/api/spec/policies/group_policy_spec.rb [hunk 2]
```diff
@@ -1,13 +1,13 @@
 RSpec.describe GroupPolicy do
+  subject { GroupPolicy }
```
Comment (hennevogel) [inline src/api/spec/policies/group_policy_spec.rb:2]: This is even already there implicitly

File: src/api/spec/policies/package_policy_spec.rb [hunk 3]
```diff
@@ -1,13 +1,13 @@
 RSpec.describe PackagePolicy do
+  subject { PackagePolicy }
```
Comment (hennevogel) [inline src/api/spec/policies/package_policy_spec.rb:2]: This is even already there implicitly

File: src/api/spec/policies/staging/staged_request_policy_spec.rb [hunk 4]
```diff
@@ -1,12 +1,12 @@
 RSpec.describe Staging::StagedRequestPolicy do
+  subject { Staging::StagedRequestPolicy }
```
Comment (hennevogel) [inline src/api/spec/policies/staging/staged_request_policy_spec.rb:2]: This is even already there implicitly

File: src/api/spec/policies/staging/workflow_policy_spec.rb [hunk 5]
```diff
@@ -1,11 +1,11 @@
 RSpec.describe Staging::WorkflowPolicy do
+  subject { Staging::WorkflowPolicy }
```
Comment (hennevogel) [inline src/api/spec/policies/staging/workflow_policy_spec.rb:2]: This is even already there implicitly

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: src/api/spec/policies/appeal_policy_spec.rb [hunk 6]
```diff
@@ -1,12 +1,12 @@
 RSpec.describe AppealPolicy do
+  subject { AppealPolicy }
```
[the remark this answers is not in the document]
Comment (hennevogel) [reply src/api/spec/policies/appeal_policy_spec.rb:2]: There are a lot of those, you get the idea...

[the remark this answers is not in the document]
Comment (eduardoj) [reply src/api/spec/policies/appeal_policy_spec.rb]: Sure, thanks! I changed it on all your comments and found a couple of them more. Please take a look again.

File: src/api/spec/policies/cloud/azure/configuration_policy_spec.rb [hunk 7]
```diff
@@ -1,11 +1,11 @@
 RSpec.describe Cloud::Azure::ConfigurationPolicy do
+  subject { described_class }
```
[the remark this answers is not in the document]
Comment (eduardoj) [reply src/api/spec/policies/cloud/azure/configuration_policy_spec.rb:2]: This is the right way of defining the test.

From the link of implicity you shared: _"If the first argument to an example group is a class, an instance of that class is exposed to each example in that example group via the subject method."_

The implicity subject takes an instance of a class. To take the class, we should use  `subject { described_class }`. This applies to all Pundit specs.

**Review by hennevogel — APPROVED** [review]

**Pull request merged**